src/components/CommandMenu/PanelCommand.tsx

```diff
@@ -1,19 +1,56 @@
-import React, { FC, useCallback, useState } from 'react'
+import React, { FC, useCallback, useRef, useState } from 'react'
 import { useSelector } from 'react-redux'
-import { css } from '../../../styled-system/css'
+import { css, cx } from '../../../styled-system/css'
 import { panelCommandRecipe } from '../../../styled-system/recipes'
 import Command from '../../@types/Command'
 import Icon from '../../@types/IconType'
 import { isTouch } from '../../browser'
 import store from '../../stores/app'
 import { executeCommandWithMulticursor } from '../../util/executeCommand'
 import fastClick from '../../util/fastClick'
+import FadeTransition from '../FadeTransition'
 
 interface PanelCommandProps {
   /** The command to execute when the button is tapped. */
   command: Command
   /** The size of the button. */
-  size?: 'small' | 'medium' | 'large' | 'xlarge'
+  size?: 'small' | 'medium'
+}
+
+interface ActiveButtonGlowImageProps {
+  size?: 'small' | 'medium'
+  isActive: boolean | undefined
+  type: 'luminosity' | 'saturation'
+}
+
+/** Glow image for active button state. */
+const ActiveButtonGlowImage: FC<ActiveButtonGlowImageProps> = ({ isActive, type }) => {
+  const nodeRef = useRef<HTMLDivElement>(null)
+  return (
+    <FadeTransition
+      type={type === 'luminosity' ? 'activeButtonGlowLuminosity' : 'activeButtonGlowSaturation'}
+      in={isActive}
+      unmountOnExit
+      nodeRef={nodeRef}
+    >
+      <div
+        ref={nodeRef}
+        className={css({
+          gridArea: 'command',
+          objectFit: 'contain',
+          objectPosition: 'center',
+          backgroundGradient: 'activeGlow',
+          borderRadius: '0px',
+          pointerEvents: 'none',
+          ...(type === 'luminosity' ? { mixBlendMode: 'luminosity' } : { mixBlendMode: 'saturation' }),
```

Comment:
I'm assuming that `mixBlendMode: type` doesn't work here because it's not statically analyzable, but we can still simplify slightly:

```ts
mixBlendMode: type === 'luminosity' ? 'luminosity' : 'saturation'
```